MAUI/PDF-Viewer/Form-Filling.md

```diff
@@ -251,6 +251,133 @@ private void PdfViewer_FormFieldValueChanged(object sender, FormFieldValueChange
 {% endhighlight %}
 {% endtabs %}
 
+## Customize the form fields programmatically
+
+Customizing form fields is crucial for controlling their appearance and behavior. By modifying properties like BackgroundColor, BorderColor, BorderWidth, and ForegroundColor, you can tailor how each form field (such as TextFormField, CheckboxFormField, RadioButtonFormField, etc.) looks within the PDF viewer. Like other form fields properties, you can undo and redo the hiding or showing actions. Additionally, when an form field is locked, it cannot be hidden. Form fields that are hidden will remain hidden during import, export, printing and saving.
+
+### Customize the backgroundcolor of the form fields
+
+The BackgroundColor property sets the color of the background within a form field. This allows you to customize the background color, which can be useful for creating forms with specific branding or visual styles in a PDF document. The following code explains how to enable the BackgroundColor for text form fields. Similarly, you can do the same for other form fields like checkboxes, radio buttons, and other form fields by modifying their respective properties. The following example illustrates how to add a background color for a TextFormField by accessing the text form field’s `BackgroundColor` property. 
```

Comment:
<img width="414" alt="image" src="https://github.com/user-attachments/assets/3340ed31-6860-44dc-92e6-f1f126c8fbbc" />

Do we have support for creating forms?